modules/pipeline/providers/cms/cms.service.go

```diff
@@ -138,6 +138,28 @@ func (s *cmsService) GetCmsNsConfigs(ctx context.Context, req *pb.CmsNsConfigsGe
 		})
 	}
 
+	// order by timeCreated
+	sortResult(results)
```

Comment:
no need sort at pipeline level, just return with input order.

Or all use two-layer map, source for first-layer and ns for second-layer.